modules/nodes-nodes-cluster-timeout-graceful-shutdown.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: CONCEPT
+[id="nodes-nodes-cluster-timeout-graceful-shutdown_{context}"]
+= About graceful node shutdown
+
+During a graceful node shutdown, the kubelet sends a termination signal to pods running on the node and postpones the node shutdown until all the pods are evicted and drained. If a node unexpectedly shuts down, the graceful node shutdown feature minimizes interruption to workloads running on these pods. 
+
+During a graceful node shutdown, the kubelet shuts down pods in two phases:
```

Comment:
please change "shuts down pods" to "terminates pods",   refer to https://kubernetes.io/blog/2021/04/21/graceful-node-shutdown-beta/